go.mod

```diff
@@ -122,11 +122,11 @@ require (
 	gorm.io/driver/postgres v1.5.0
 	gorm.io/gorm v1.25.0
 	helm.sh/helm/v3 v3.11.3
-	k8s.io/api v0.26.4
+	k8s.io/api v0.27.1
 	k8s.io/apimachinery v0.27.1
 	k8s.io/apiserver v0.26.4
 	k8s.io/cli-runtime v0.26.4
-	k8s.io/client-go v0.26.4
+	k8s.io/client-go v0.27.1
```

Comment:
I think this broke the build. Is there any chance to stay with 0.26? If yes, that would be great, if not, we need to apply the changes that come with 0.27 to our codebase.

Reply:
no, netpol-analyzer's build will fail with  k8s.io/client-go v0.26.4

Reply:
I would suggest then to open a separate PR where we update this dep to v0.27 and make all the CI-checks green. After that we can rebase this PR and continue here

Reply:
We succeeded to compile `netpol-analyzer ` with `openshift/api` version that compiles with `	k8s.io/client-go v0.26.4`.
committed the new `go.mod`

however, in order not to break the build : I've added these lines to the end of the file :
```
exclude k8s.io/client-go v0.27.1
exclude k8s.io/api v0.27.1
```
so it will not be updated automatically by `go mod tidy` and would not affect code sections which are not `np-guard`

Reply:
committed another fix to `go.mod ` without all the `exclude` statements (keeping the original 	k8s.io/client-go v0.26.4)